### PR TITLE
Improved feature-bar styling for the repo page

### PR DESCRIPTION
**Pull request by txsmith**
screenshot from 2016-12-12 16-16-21

screenshot from 2016-12-12 16-16-30

### CODE REVIEW

File: src/projects/project-overview-page.html [hunk 1]
```diff
@@ -83,24 +86,91 @@
...
+        box-shadow: -20px 0 20px 0px rgb(255, 255, 255);
       }
     </style>
 
     <app-route route="{{route}}" pattern="/:page" data="{{page}}"></app-route>
 
     <feature-bar selected-feature={{page.page}}>
       <div class="content">
-        <h2 class="title">[[project.owner]] / [[project.name]]</h2>
-        <span class="description">[[project._actual.description]]</span>
```
Comment (eanker) [inline src/projects/project-overview-page.html:102]: I see you deleted the project description, was this a deliberate decision?
We might want to add this again to the header bar.

Comment (txsmith) [reply src/projects/project-overview-page.html:102]: screenshot from 2017-02-13 11-00-22

Comment (txsmith) [reply src/projects/project-overview-page.html:102]: Note that markdown is not supported on GitHub either

Comment (eanker) [reply src/projects/project-overview-page.html:102]: We can always add that as a feature later. I think this is good enough.

**Pull request merged**